pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/vedlegg/OpplysningerBruktIBeregningenAlder.kt

```diff
@@ -187,6 +189,15 @@ val vedleggOpplysningerBruktIBeregningenAlder =
                         English to "Your accumulated pension capital",
                     )
                 }
+                
+                paragraph {
```

Comment:
Blir include condition rett for denne no? Ref doksys sin `if (regelType == "AP2011" && metode == "FOLKETRYGD" && tilleggspensjon == "")`

Reply:
Ah. Det blir den ikke. Må ha blanda noen tekster når jeg implementerte dette. Den logikken hører til innholdet under, ikke tittelen og avsnittet. Skal rette opp.

Reply:
Aha. Det er ikke bare det som er feil. Det som er forvirrende her er at tittelen er den samme, så det er egentlig if(xyz) { tittel, tekst} else if (zyx){tittel, tekst ... tabell}.